raster/src/main/scala/geotrellis/raster/render/ascii/NumericEncoder.scala

```diff
@@ -0,0 +1,91 @@
+package geotrellis.raster.render.ascii
+
+
+import geotrellis.raster.{Tile, isNoData}
+
+import spire.syntax.cfor._
+
+import scala.collection.mutable.ArrayBuffer
+import scala.math.BigDecimal
+
+import java.util.Locale
+
+
```

Comment:
There are no headers, mb it makes sense to do it in a separate PR before 2.0 release, after all issues would be closed.

Reply:
👍